doc/classes/String.xml

```diff
@@ -302,7 +326,7 @@
 			<argument index="1" name="from" type="int" default="0">
 			</argument>
 			<description>
-				Find the first occurrence of a substring but search as case-insensitive, return the starting position of the substring or -1 if not found. Optionally, the initial search index can be passed.
+				Finds the first occurrence of a substring but searches as case-insensitive, returns the starting position of the substring or -1 if not found. Optionally, the initial search index can be passed.
```

Comment:
- comma to period. "Returns..."
- "searches as case-insensitive" isn't good grammar.  How about

"Finds the first occurrence of a substring, ignoring case.